app/services/payment_providers/stripe_service.rb

```diff
@@ -110,6 +111,19 @@ def handle_event(organization:, event_json:)
             metadata: event.data.object.metadata.to_h.symbolize_keys,
           )
         result.raise_if_error! || result
+      when 'customer.updated'
+        payment_method_id = event.data.object.invoice_settings.default_payment_method || event.data.object.default_source
```

Comment:
Layout/LineLength: Line is too long. [121/120] (https://rubystyle.guide#max-line-length)